tox.ini

```diff
@@ -1,10 +1,10 @@
 [tox]
-envlist = py{27,36,37}-sphinx{16,17,18,20,21,22,23,24,30}
+envlist = py{36,37,38}-sphinx{18,20,21,22,23,24,30,31,32}
```

Comment:
We could add 3.9 as well here that was just released.

Reply:
Isn't in travis yet https://docs.travis-ci.com/user/languages/python/#python-versions